apps/wallet-dashboard/components/Dialogs/Assets/views/DetailsView.tsx

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { ExplorerLinkType, useNftDetails, Collapsible } from '@iota/core';
+import {
+    Button,
+    ButtonType,
+    Header,
+    KeyValueInfo,
+    VisualAssetCard,
+    VisualAssetType,
+} from '@iota/apps-ui-kit';
+import Link from 'next/link';
+import { formatAddress } from '@iota/iota-sdk/utils';
+import { Layout, LayoutBody, LayoutFooter } from '../../Staking/views/Layout';
+import { IotaObjectData } from '@iota/iota-sdk/client';
+import { ExplorerLink } from '@/components/ExplorerLink';
+import { useCurrentAccount } from '@iota/dapp-kit';
+
+interface DetailsViewProps {
+    asset: IotaObjectData;
+    handleClose: () => void;
+    handleSend: () => void;
+}
+
+export function DetailsView({ handleClose, asset, handleSend }: DetailsViewProps) {
+    const account = useCurrentAccount();
+
+    const senderAddress = account?.address ?? '';
+    const objectId = asset.objectId;
+
+    const {
+        nftName,
+        nftImageUrl,
+        nftMeta,
+        ownerAddress,
+        isTransferable,
+        metaKeys,
+        metaValues,
+        formatMetaValue,
+        isContainedInKiosk,
+        kioskItem,
+    } = useNftDetails(objectId, senderAddress);
+
+    function handleMoreAboutKiosk() {
+        window.open('https://wiki.iota.org/', '_blank');
+    }
+
+    function handleMarketplace() {
+        window.open('https://wiki.iota.org/', '_blank');
+    }
+
+    return (
+        <Layout>
+            <Header title="Asset" onClose={handleClose} titleCentered />
+            <LayoutBody>
+                <div className="flex w-full flex-col items-center justify-center gap-xs">
+                    <div className="w-[172px]">
+                        <VisualAssetCard
+                            assetSrc={nftImageUrl}
+                            assetTitle={nftName}
+                            assetType={VisualAssetType.Image}
+                            altText={nftName || 'NFT'}
+                            isHoverable={false}
+                        />
+                    </div>
+                    <ExplorerLink type={ExplorerLinkType.Object} objectID={objectId}>
+                        <Button type={ButtonType.Ghost} text="View on Explorer" />
+                    </ExplorerLink>
+                    <div className="flex w-full flex-col gap-md">
+                        <div className="flex flex-col gap-xxxs">
+                            <span className="text-title-lg text-neutral-10">{nftMeta?.name}</span>
```

Comment:
we need a dark theme text here
<img width="371" alt="Screenshot 2024-11-26 at 16 09 02" src="https://github.com/user-attachments/assets/f029f16e-e86e-4979-8d60-c67938a6bdd1">